javascript/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.0.0",
   "main": "./stimulus_reflex.js",
   "scripts": {
-    "prettier-check": "yarn run prettier --check ./stimulus_reflex.js"
+    "prettier-check": "yarn run prettier-standard --check ./stimulus_reflex.js"
```

Comment:
@hopsoft shouldn’t need to update the action. It just executed this script. Since you’ve updated it, it should continue working.